### PR TITLE
Update jepa finetuning:

### LINKED ISSUE #1956: Move JEPA finetuning eval to IASI
Move JEPA finetuning eval to IASI and remove encoder parameters

### PR DESCRIPTION
**Pull request by sophie-xhonneux**
## Description

Update the jepa finetuning configs to use IASI by default

## Issue Number

Closes #1956

Is this PR a draft? Mark it as draft.

## Checklist before asking for review

-   [x] I have performed a self-review of my code
-   [ ] My changes comply with basic sanity checks:
      - I have fixed formatting issues with `./scripts/actions.sh lint`
      - I have run unit tests with `./scripts/actions.sh unit-test`
      - I have documented my code and I have updated the docstrings.
      - I have added unit tests, if relevant
-   [ ] I have tried my changes with data and code:
      - I have run the integration tests with `./scripts/actions.sh integration-test`
      - (bigger changes) I have run a full training and I have written in the comment the run_id(s): `launch-slurm.py --time 60`
      - (bigger changes and experiments) I have shared a hegdedoc in the github issue with all the configurations and runs for this experiments
-   [ ] I have informed and aligned with people impacted by my change:
    - for config changes: the MatterMost channels and/or a design doc
    - for changes of dependencies: the MatterMost software development channel

### CODE REVIEW

**Review by shmh40 — APPROVED** [review]
just changed the end date of training period -- unless you were saving september for something special?

And so Disabled actually works as a type for the loss now? we should use this in future?
